packages/servicenow_cmdb/data_stream/event/manifest.yml

```
@@ -0,0 +1,498 @@
+title: Event
+type: logs
+streams:
+  - input: cel
+    title: ServiceNow tables
+    description: Collect table logs from ServiceNow CMDB instance via API.
+    template_path: cel.yml.hbs
+    enabled: false
+    vars:
+      - name: url
+        type: url
+        title: URL
+        description: URL of ServiceNow instance. It should follow the given format, https://<instance_id>.service-now.com.
+        required: true
+        show_user: true
+      - name: username
+        type: text
+        title: Username
+        description: The username to be used with Basic Auth headers.
+        multi: false
+        required: true
+        show_user: true
+      - name: password
+        type: password
+        title: Password
+        description: The password to be used with Basic Auth headers.
+        multi: false
+        required: true
+        show_user: true
+        secret: true
+      - name: initial_interval
+        type: text
+        title: Initial Interval
+        multi: false
+        required: true
+        show_user: true
+        default: 24h
+        description: How far back to pull the logs from ServiceNow CMDB API. Supported units for this parameter are h/m/s.
+      - name: interval
+        type: text
+        title: Interval
+        description: Duration between requests to the ServiceNow API. Supported units for this parameter are h/m/s.
+        default: 5m
+        multi: false
+        required: true
+        show_user: true
+      - name: batch_size
+        type: text
+        title: Batch Size
+        multi: false
+        required: true
+        show_user: true
+        description: Batch size for the response of the ServiceNow API, with a maximum limit of 10,000.
+        default: 10000
+      - name: table_name
+        type: text
+        title: Table Name
+        description: Table name to fetch logs for. Please enter the table name in lowercase with underscores. For example- cmdb_ci_infra_service for the Infrastructure Service table. Refer this [link](https://<your_instance_id>.service-now.com/now/nav/ui/classic/params/target/sys_db_object_list.do) for the exact table names.
```

```suggestion
        description: Table name to fetch logs for. Please enter the table name in lowercase with underscores. For example, cmdb_ci_infra_service for the Infrastructure Service table. Refer this [link](https://<your_instance_id>.service-now.com/now/nav/ui/classic/params/target/sys_db_object_list.do) for the exact table names.
```
Note that the link will be broken. Please find another way to communicate to the user what it is that you are trying to get across.